CHANGELOG.md

```diff
@@ -6,6 +6,8 @@ Currently the versioning policy of this project follows [Semantic Versioning v2.
 
 ## Unreleased - 2017-??-??
 
+## 3.1.0 - 2017-10-23
```

Comment:
This date was too optimistic ... Should we change?

Reply:
OK I will.